Products/CMFPlone/browser/navigation.py

```diff
@@ -240,8 +240,8 @@ def breadcrumbs(self):
         if IHideFromBreadcrumbs.providedBy(context):
             return base
 
-        if base:
-            item_url = '%s/%s' % (base[-1]['absolute_url'], name)
+        # if base:
+        #     item_url = '%s/%s' % (base[-1]['absolute_url'], name)
```

Comment:
If code is no longer needed please just remove it. Git keeps it anyway, if someone is interested in it.